### PR TITLE
Q93 error?

### PR DESCRIPTION
**Pull request by promeraq**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [ ] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [ x ] I have made small correction/improvements

### Changes / Instructions

_Add instructions to me, please type here, thanks_

With animal.shift() we remove jaguar before the last line, where we remove eagle and print it. With animal.pop() we remove eagle and then remove and print jaguar

### CODE REVIEW

File: javascript/javascript-quiz.md [hunk 1]
```diff
@@ -1219,8 +1219,8 @@ console.log(animals.shift()); //Prints jaguar
 
 - [ ] `animals.filter(e => e === "jaguar");`
 - [ ] `animals.reverse();`
-- [x] `animals.shift();`
-- [ ] `animals.pop();`
+- [ ] `animals.shift();`
+- [x] `animals.pop();`
```
Comment (Ebazhanov) [inline javascript/javascript-quiz.md:1223]: thanks @promeraq for your PR, but i just checked it in console, as result 
image

**Review by Ebazhanov — COMMENTED** [review]
see comments

File: javascript/javascript-quiz.md (hunk 1 above)
Comment (promeraq) [reply javascript/javascript-quiz.md:1223]: Q93. Which line COULD YOU ADD to this code to print "jaguar" to the console? 
You are just checking the statement in console, but have to add a missing line.

Comment (Ebazhanov) [reply javascript/javascript-quiz.md:1223]: you are right, LGTM

**Comment by Ebazhanov** [comment]
@all-contributors please add @promeraq for design, code

**Pull request merged**